Nesteo.Server/Controllers/Api/NestingBoxesController.cs

```diff
@@ -69,5 +71,14 @@ public async Task<ActionResult<NestingBoxPreview>> GetNestingBoxPreviewByIdAsync
 
             return nestingBoxPreview;
         }
+
+        /// <summary>
+        /// Return all inspections for a nesting box
+        /// </summary>
+        [HttpGet("{id}/inspections")]
+        public IAsyncEnumerable<Inspection> GetInspectionsByBoxIdAsync(string id)
```

Comment:
Becasue we've written-out "NestingBox" everywhere else, I think `GetInspectionsForNestingBoxIdAsync` would be more appropriate